### DISCUSSION TITLE
add Category and tag to upload media

### DISCUSSION THREAD

**Discussion by btsavb** [mediacms-io/mediacms#812] [Feature Requests]
Hello,
After upload of multiple media you have to edit each of them to assign categories and tags.
Is it possible to add a featured that you can assign global categories or/and tags directly at the upload of files?
Thanks

**Comment by calamza** [comment]
I have a similar situation
I upload a lot of media file (videos and audios) but I really need to tag or categorize them. I think maybe with the API but I don't undertand how.
Did you solve it?
